lib/stacker/stack.rb

```diff
@@ -141,8 +160,77 @@ def update options = {}
       end
     end
 
+    def describe_change_set
+      retries = 6
+      changes = []
+      while changes.empty?
+        resp = region.client.describe_change_set(
+          change_set_name: change_set,
+          stack_name: name
+        )
+        changes = resp.changes
+        if changes.empty?
+          raise CannotDescribeChangeSet.new 'Empty change set' if retries == 0
+          retries -= 1
+          sleep 1
```

Comment:
Could you explain the need for sleep and retries? I haven't used change sets myself, so just unclear why the need for multiple attempts.